src/main/java/seedu/quotesify/book/BookList.java

```diff
@@ -0,0 +1,194 @@
+package seedu.quotesify.book;
+
+import org.json.simple.JSONArray;
+import seedu.quotesify.author.Author;
+import seedu.quotesify.commands.Command;
+import seedu.quotesify.exception.QuotesifyException;
+import seedu.quotesify.lists.QuotesifyList;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+public class BookList extends QuotesifyList<Book> {
```

Comment:
Should the arrows demonstrate associations? The arrows in the attached screenshot seem a bit off from the module arrows. I think you may need to change the type of arrow.
![class_notation](https://user-images.githubusercontent.com/55688719/97387628-5ec46600-1911-11eb-8b8a-4097be44c8b8.PNG)